Gemfile.lock

```diff
@@ -596,7 +601,7 @@ GEM
     zeitwerk (2.6.6)
 
 PLATFORMS
-  ruby
+  x86_64-linux
```

Comment:
🤔 je suis pas sûr de savoir comment cette valeur est utilisée, mais est-ce qu'on a vraiment besoin de la changer ?

Reply:
Pour une raison qui m'échappe, on avait `nokogiri (1.13.9-x86_64-linux)` dans la précédente PR. J'ai viré ce forçage de plateforme dans ce nouveau commit 203905d603a06879a529992d3832355ed13a239b !